src/libs/antares/study/parts/renewable/container.cpp

```diff
@@ -38,7 +38,7 @@ namespace Antares
 {
 namespace Data
 {
-PartRenewable::PartRenewable() : list(), clusters(nullptr), clusterCount((uint)-1)
+PartRenewable::PartRenewable() : list(), clusters()
```

Comment:
Do we really need the call to default `clusters()` constructor ?

Reply:
Right, I also removed the call to other default ctors.